py-polars/polars/series/series.py

```diff
@@ -869,6 +869,12 @@ def __copy__(self) -> Self:
     def __deepcopy__(self, memo: None = None) -> Self:
         return self.clone()
 
+    def __contains__(self, item) -> bool:
+        try:
```

Comment:
Can you add a 

```python
# `TODO! optimize via `is_in` and `SORTED` flags`
```

Reply:
Btw do you know there is a special feature in GitHub for concrete change suggestions in review comments?